api/nakadi-event-bus-api.yaml

```diff
@@ -1152,3 +1550,108 @@ definitions:
           should have a description.
     required:
       - publishing_status
+
+  PaginationLinks:
+    description: contains links to previous and next pages of items
+    type: object
+    properties:
+      prev:
+        $ref: '#/definitions/PaginationLink'
+      next:
+        $ref: '#/definitions/PaginationLink'
+
+  PaginationLink:
+    description: URI identifying another page of items
+    type: object
+    properties:
+      href:
+        type: string
+        format: uri
+        example: '/subscriptions?offset=20&limit=10'
+
+parameters:
+  EventTypeName:
+    name: name
+    in: path
+    description: EventType name to get events about
+    type: string
+    required: true
+
+  BatchLimit:
+    name: batch_limit
```

Comment:
Unfortunately we are not renaming this not to keep compatibility with low level parameter names.